docs/contributing.rst

```diff
@@ -23,3 +23,10 @@ Pull requests are automatically built with `Travis-CI <https://travis-ci.org/one
     * Documentation builds successfully (including spell-check)
 
 See :ref:`getting in touch <getting_in_touch>` for more guidance on contributing.
+
+Running the tests
+--------------------
+
+To run the tests for the library, you must have :ref: `rabbitmq installed <installation>` and running locally. Once rabbitmq is up, run the tests::
```

Comment:
It is possible to test against a non-local rabbit actually, though not with the Makefile. You could make this slightly more correct by switching the order of the two sentences:

``` rst
There is a Makefile with convenience commands for running the tests. To run them locally you must have RabbitMQ :ref:`installed <installation>` and running, then call::

    make test
```